packages/flutter/lib/src/material/ink_well.dart

```diff
@@ -64,6 +64,79 @@ abstract class InteractiveInkFeature extends InkFeature {
     _color = value;
     controller.markNeedsPaint();
   }
+
+  /// Draws an ink splash or ink ripple on the passed in [Canvas].
+  ///
+  /// The [transform] argument is the [Matrix4] transform that typically
+  /// shifts the coordinate space of the canvas to the space in which
+  /// the ink circle is to be painted.
+  ///
+  /// [center] is the [Offset] from origin of the canvas where center
+  /// of the circle is drawn
+  ///
+  /// Using [paint] you can specify properties like color, strokewidth, colorFiler, etc
+  /// for painting the circle
+  ///
+  /// [radius] is the radius of circle to be drawn on canvas
+  ///
+  /// [clipCallback] is required by ink effects to obtain the rectangle for the effect,
+  /// if [clipCallback] is null, no clipping is performed and only the ink circle is painted
+  ///
+  /// [customBorder] is a [ShapeBorder] which can be used if you want to clip
```

Comment:
- Tailor the doc to be specific to the function (painting ink circles)
- End sentence with period
```suggestion
  /// [customBorder] is a [ShapeBorder] which can be used to clip the 
  /// ink circle with a custom path other than a [Rect] or [RRect]. 
```